cgmanifest.json

```diff
@@ -1,5 +1,7 @@
 {
-  "Registrations": [
+  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+  "version": 1,
+  "registrations": [
```

Comment:
Looks like `jq` (and other tools probably as well) are case sensitive, so please also update the word "Registration" in these files:
- `.github/workflows/validate-cg-manifest.sh`,
- `toolkit/scripts/download_spec_sources.sh`,
- `toolkit/scripts/update_cgmanifest.py`.